java/IBAutomater/src/ibautomater/WindowEventListener.java

```diff
@@ -385,5 +395,61 @@ private boolean HandleFinancialAdvisorWarningWindow(Window window, int eventId)
 
         return false;
     }
+
+    private boolean HandleExitSessionSettingWindow(Window window, int eventId) throws Exception {
+        if (eventId != WindowEvent.WINDOW_ACTIVATED) {
+            return false;
+        }
+
+        String title = Common.getTitle(window);
+
+        if (title != null && title.contains("Exit Session Setting")) {
+            LocalDateTime time = java.time.LocalDateTime.now();
+
+            // disable logout/restart by setting logoff/restart time to the next day
```

Comment:
nit - comment is off
Did we get `time.minusMinutes(10);` from testing or is it documented?

Reply:
The comment is actually correct, perhaps not clear enough, this is what happens:
- at `3:07 AM` this window pops up:
![image](https://user-images.githubusercontent.com/7236689/78011609-17a6d000-7344-11ea-912e-b0e64ae05dfc.png)
- the `Cancel` button just closes the window, but will open again after a few seconds
- to prevent the logout, the only option is to move the time away from the current time: by setting the time to `2:57 AM` (10 minutes before the current time) we are basically telling IBGateway to not perform the logout until tomorrow at `2:57 AM`; a few minutes before that time the window will open again and we will repeat the process

The 10 minutes value is an arbitrary choice, any positive value should be ok.

Reply:
Got it 👍